src/ontology/templates/assays.tsv

```diff
@@ -240,6 +240,7 @@ OBI:0002112	small RNA sequencing assay		http://purl.obolibrary.org/obo/IAO_00001
 OBI:0002113	bromouridine labeling and sequencing assay		http://purl.obolibrary.org/obo/IAO_0000122	Bru-seq	An RNA-seq assay to identify spans of nascent transcription in the genome through isolation of recent bromouridine (Bru) labelled RNAs.	PMC:4009065		Jason Hilton|Chris Stoeckert					subclass		RNA-seq assay													
 OBI:0002114	bromouridine pulse-chase and sequencing assay		http://purl.obolibrary.org/obo/IAO_0000122	BruChase-seq	An RNA-seq assay to identify RNA populations of specific ages through isolation of RNAs first labelled with bromouridine (Bru) followed by chasing in uridine for different periods of time.	PMC:4009065		Jason Hilton|Chris Stoeckert					subclass		RNA-seq assay													
 OBI:0002115	cytometry time of flight assay			CyTOF|mass cytometry assay	A cytometry assay in which the presence of molecules of interest on or in cells is indicated by binding of antibodies labeled with rare earth element tags which are detected by time-of-flight mass spectrometry.	PMID:26190063	Whole human blood samples were incubated with lipopolysaccharide (LPS) to examine dose-dependent signaling responses within the toll-like receptor 4 (TLR4) pathway. After LPS stimulation, rare earth metal tagged antibodies against phenotypic markers were used to stain the blood sample (Nd143-CD45RA, Nd145-CD4, Er170-CD3, Sm152-TCR__, Pr141-CD7, Nd146-CD8, Nd142-CD19). A Fluidigm CyTOF version 1 instrument equipped with CyTOF software version 5.1.648 was used to measure and analyze the levels of antibody staining refelctive of the level of expression of those surface receptors.	ImmPort					subclass	logical definition does not capture ""time of flight"" part	cytometry assay		mass spectrometry assay						rare earth metal atom					
+OBI:0002115				CyTOF|mass cytometry assay				Michelle Giglio|Suvarna Nadendla|Rebecca Jackson			CFDE	https://github.com/obi-ontology/obi/issues/1402	subclass															
```

Comment:
Something is wrong here. There are two rows with ID "OBI:0002115", and the second is missing a label. We should have a test to catch this.